skins/cat17/templates/Warning_Page.html.twig

```diff
@@ -0,0 +1,20 @@
+	<div class="container">
+		<div class="row">
+			<div class="content col-xs-12 col-sm-10 col-md-8">
+				<h2>{$ 'error_page_header' | trans $}</h2>
+    			{% if message is defined %}
+					<p>
+						{$ message | trans | raw $}
```

Comment:
The message is already translated in `Confirm_Subscription.twig`. Translating it again here won't change it, (because the translation is not found), but it's a potential future error. I'd remove the `| trans` here, because the one in  `Confirm_Subscription.twig` also knows that the message comes from the validation and uses the right message file.